DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: algorithmia
 Type: Package
 Title: Allows you to Easily Interact with the Algorithmia Platform
-Version: 0.1.0
+Version: 0.2.0
```

Comment:
This was never published, right? so we can keep the same version?

Reply:
0.1.0 was published, 0.2.0 was - I pushed that to master accidentally so it should just null in the merge